webgrid/filters.py

```diff
@@ -177,9 +179,11 @@ def format_invalid(self, exc, col):
             str(exc)
         )
 
-    def new_instance(self):
+    def new_instance(self, dialect=None):
```

Comment:
We should have a comment here directing developers overriding new_instance to include **kwargs in their method signature, to prevent future breakage.